src/suricata.c

```diff
@@ -169,10 +172,31 @@
 
 #include "tmqh-packetpool.h"
 
-#include "util-proto-name.h"
+#include "util-byte.h"
```

Comment:
are these large include changes really needed? Or are they a artifact of cherry-picking?

Reply:
From cherry-picking .... the harmful includes were removed but there are probably extra includes.

Reply:
can you minimize it, we need backports to be as clean as possible